examples/file-upload/README.md

```diff
@@ -0,0 +1,37 @@
+# @loopback/example-file-upload
+
+An example application to demonstrate file uploads for LoopBack 4
+
+## Summary
+
+This application comes with a controller that exposes `/file-upload` endpoint
+that accepts `multipart/form-data` based file uploads.
+
```

Comment:
I think it would be useful to mention:
- the files that you've created that are related to this upload example, possibly just link to the controller and service file. 
- use of the Express middleware multer